cpp-client/tests/test_util.cc

```diff
@@ -114,7 +115,11 @@ TableMakerForTests TableMakerForTests::create() {
 }
 
 Client TableMakerForTests::createClient() {
-  std::string connectionString("localhost:10000");
+  const char* hostptr = std::getenv("DH_HOST");
+  const char* portptr = std::getenv("DH_PORT");
```

Comment:
super-mega-nit: because of how C++ binding works I'd prefer

```
const char *hostptr
const char *portptr
```

i.e. the * being closer to the variable name